images/runtime/python/template.Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN apt-get update \
     && apt-get upgrade -y \
     && apt-get install -y --no-install-recommends \
         xz-utils \
+        gcc \
```

Comment:
Why do we need to install this library?

Reply:
I'll add a one line comment to explain a bit. gcc was added because `pip install viztracer` was returning gcc command not found during building stage of runtime image. This one was added here to resolve that issue.